src/components/CubeListNavbar.js

```diff
@@ -108,6 +108,35 @@ const UploadBulkModal = ({ isOpen, toggle }) => {
 
 const UploadBulkModalItem = withModal(DropdownItem, UploadBulkModal);
 
+const UploadBulkReplaceModal = ({ isOpen, toggle }) => {
+  const { cubeID } = useContext(CubeContext);
+  return (
+    <Modal isOpen={isOpen} toggle={toggle} labelledBy="uploadReplacementModalTitle">
+      <ModalHeader id="uploadReplacementModalTitle" toggle={toggle}>
+        Bulk Upload - Replace with CSV File Upload
+      </ModalHeader>
+      <CSRFForm method="POST" action={`/cube/bulkreplacefile/${cubeID}`} encType="multipart/form-data">
+        <ModalBody>
+          <p>Acceptable files are .csv files with the exact format as our .csv export.</p>
```

Comment:
Can we add a new sentence before this one stating that this upload will *replace* the current contents of your cube?